.github/actions/check_for_stc/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Ensure STC is installed with Pyccel'
+inputs:
+  editable:
+    description: 'editable installation'
```

Comment:
It might be easier to add a default value for this

Reply:
Okay, but what should be the default value? How about using `default: '-e'` or `editable`?

Reply:
I think the default should be the non-editable case as this is used more often